### PR TITLE
Add Payments block refresher - Refresh when product is added or removed from Order

### LINKED ISSUE #21293: BO - Order details - Payment block not refreshed after adding product
#### Describe the bug

On BO, when adding/updating a product on an already paid order, the message in Payment block "Warning €XX paid instead of €XX" is not added/updated immediately. A refresh is needed (on Legacy refresh wasn't needed).

Please note that when a voucher is added, the Payment block is automatically updated without refresh.

#### Expected behavior

Payment block should be updated each time an update is made on the order.

#### Steps to Reproduce

0. Have a paid order
1. Go to BO > Orders > On the paid orders
2. Add or Edit a product on the order
3. The "Warning" message in payment block is not displayed
4. Refresh the page : Warning message is now displayed.

**Screenshots**

On screenrecord : I add a product, Payment block is not updated. I refresh : The "warning" message is now updated in Payment block.

https://drive.google.com/file/d/1A5gryu4jr0oSdI7B4Oy_UeulJeNVp39Z/view?usp=sharing

#### Additional information

* PrestaShop version: 1770rc1build1
* PHP version: N/A

### PR DESCRIPTION
**Pull request by sowbiba**
## DONT MERGE BEFORE 1770 RELEASE

| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.7.x
| Description?  | Add component to refresh alert for already paid order when a product is added to or removed from Order
| Type?         | bug fix
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | Fixes #21293
| How to test?  | Check the __How to reproduce__ section of the issue

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/22059)

### CODE REVIEW

**Review by Progi1984 — DISMISSED** [review]

File: src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/payments_alert.html.twig
Comment (atomiix) [inline src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/payments_alert.html.twig]:
```suggestion
{% endif %}

```

**Review by atomiix — DISMISSED** [review]

File: admin-dev/themes/new-theme/js/pages/order/view/order-payments-refresher.js
Comment (matks) [inline admin-dev/themes/new-theme/js/pages/order/view/order-payments-refresher.js]: Maybe we should handle error (HTTP 400 if order ID is bad or hTTP 500 is server fails) ?

**Review by matks — COMMENTED** [review]
Suggestion for improvement ;) otherwise nice job!

**Review by matks — APPROVED** [review]

**Review by atomiix — APPROVED** [review]

File: admin-dev/themes/new-theme/js/pages/order/view/order-payments-refresher.js
Comment (jolelievre) [inline admin-dev/themes/new-theme/js/pages/order/view/order-payments-refresher.js:31]: I'm just a bit disturbed by this naming (in route and urls as well) It feels like you are updating the list of payments but really it only updates the Payment Alert
I don't block this since it already has two approvals but I really think naming could be improved, and since QA is quite busy for now you should have time to fix it

The overall PR is fine besides this naming issue

**Comment by khouloudbelguith** [comment]
Hi @sowbiba,

It is ok :heavy_check_mark: 
https://drive.google.com/file/d/1epJPfPI63EIVh0mbAoLXPKLzI2xQScbz/view?usp=sharing

Thanks!

**Comment by Progi1984** [comment]
## DONT MERGE BEFORE 1770 RELEASE

**Comment by Progi1984** [comment]
Thanks @sowbiba & @khouloudbelguith

**Pull request merged**
